src/module.py

```diff
@@ -0,0 +1,46 @@
+'''
+Contains the nn.Modules which compose the networks. This includes modules 
+for preprocessing speech and text, the transformer & RNN encoder/decoder, & post
+processing modules for text and speech.
+'''
+
+class SpeechPrenet(nn.Module):
+    # TODO: Fill in from TTS repo :) 
+    def __init__(self):
+        super(SpeechPrenet, self).__init__()
+
+class SpeechPostnet(nn.Module):
+    # TODO: Fill in from TTS repo :) 
+    def __init__(self):
+        super(SpeechPostnet, self).__init__()
+
+class TextPrenet(nn.Module):
+    # TODO: Fill in from TTS repo :) 
+    def __init__(self):
+        super(TextPrenet, self).__init__()
+
+class TextPostnet(nn.Module):
+    # TODO: Fill in from TTS repo :) 
+    def __init__(self):
+        super(TextPostnet, self).__init__()
+
+class TransformerEncoder(nn.Module):
+    # TODO: Fill in from TTS repo :) 
+    def __init__(self):
+        super(TransformerEncoder, self).__init__()
+
+class TransformerDecoder(nn.Module):
+    # TODO: Fill in from TTS repo :) 
+    def __init__(self):
+        super(TransformerDecoder, self).__init__()
+
+class RNNEncoder(nn.Module):
+    # TODO: Write this
+    def __init__(self):
+        super(RNNEncoder, self).__init__()
+
+class RNNDecoder(nn.Module):
+    # TODO: Write this
+    def __init__(self):
+        super(RNNDecoder, self).__init__()
+
```

Comment:
How are these different from the classes in `src/network.py`?

Reply:
Network will be the e2e components, so they will wrap these modules inside of them, exposing access where necessary.